### ISSUE TITLE
Rule(rule) method does not modify rule with append/prepend

### ISSUE THREAD

**Issue by DenyVeyten** [postcss/postcss#1418]
postcss: 8.0.5

Trying to add Declaration to each rule in css with Rule method, append/prepend does not modify the node (rule.nodes remain the same)

The code leaves css unmodified:
```
module.exports = () => ({
  postcssPlugin: 'postcss-plugin',
  Rule(rule, { Declaration }) {
    rule.append(new Declaration({ prop: 'color', value: `green` }));
  },
});

module.exports.postcss = true;
```

PR with example https://github.com/DenyVeyten/postcss-deadcss/pull/1

**Comment by ai** [comment]
Fixed 1dc16fa

**Comment by ai** [comment]
The fix was released in 8.0.6
